example-code/snippets/src/backend/IModelTestUtils.ts

```diff
@@ -34,10 +34,10 @@ export class KnownTestLocations {
 
 export class IModelTestUtils {
 
-  private static _connectClient: ContextRegistryClient | undefined;
-  public static get connectClient(): ContextRegistryClient {
+  private static _connectClient: ITwinAccessClient | undefined;

```

Comment:
nit: can we rename this?
idk where this example code gets used besides maybe our docs, but dont think we should be exposing "connect" anywhere